carousel/carousel.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Plugin Name:       Carousel
+ * Description:       Example block scaffolded with Create Block tool.
```

Comment:
Please update the description

Reply:
Thanks for catching these, @nate-allen! I'll get both fixed up soon.